### PR TITLE
Updating project profiles

### PR DESCRIPTION
**Pull request by LauraUSDS**
_Laura to double-check text, images, formatting on:_
- [x] each individual profile page
- [x] each project carousel card
- [x] full projects page

### CODE REVIEW

**Comment by LauraUSDS** [comment]
@drew-usds @lisac-usds 

**How hard is it to change the header text for the sidebar numbers?**
I figure this is constant across all project profile pages. It currently says "The Impact" which is odd now that some project profiles contain a "The Impact" section in the body. Propose changing to "By the numbers" or something similar.

**Mostly out of curiosity, can someone explain how certain project pages are in the [/_projects folder](https://github.com/usds/website/tree/staging/_projects) but don't appear on the website?**
Specifically:
- 0600_disability-compensation-application.md
- 0800_personalized-dashboard.md
- 0900_disability-appeals-status.md
- 1400_gi-bill-comparison-tool.md
I know we don't know the decision process of why they aren't live anymore, but I'm curious if it's a fluke. Also asking how we can archive project pages, since I'm sure that'll come up and then maybe I won't have to bother y'all. ;)

CC @bernars-usa for FYI

**Comment by lisac-usds** [comment]
Hi @LauraUSDS ! Some quick thoughts - would you please let me know whether I should implement any of these on this `staging` branch?

1. How hard is it to change the header text for the sidebar numbers?
Possibly a one-line change: 
https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_includes/_template--project-page.html#L9-L12

2. certain project pages are in the /_projects folder but don't appear on the website?
This might be respecting a `hide` flag, eg
   - https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_projects/0600_disability-compensation-application.md#L8-L12
   - https://github.com/usds/website/blob/ef2b53f0556c34057bc7cf99962cf1dfbea4c607/_projects/1400_gi-bill-comparison-tool.md#L8-L12

3. how we can archive project pages
Might have a few options. How about we move them into the existing [_projects-drafts](https://github.com/usds/website/tree/master/_projects-drafts) directory (and maybe rename)? Or we could create a standalone `_projects-archives` directory. Those approaches would be low level of effort.  We could also consider adding something like an `archived` flag (similar to `hide`).

File: _projects/0400_blue-button-2.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ banner_file_mobile: banner--people-projects-sm.svg
 project_page: true
 
 # Carousel (Edit this)
-carousel_title: "Modernizing health data access with Medicare APIs"
-carousel_summary: "We worked with CMS to build an API that enables beneficiaries with different needs to grant access to developers that can help them monitor for drug conflicts, refill prescriptions, and track progress towards desired healthcare outcomes."
-carousel_image_name: project-hhs-blue-button-card.jpg
+carousel_title: "Empowering beneficiares with their own healthcare data"
```
Comment (lisac-usds) [inline _projects/0400_blue-button-2.md]:
```suggestion
carousel_title: "Empowering beneficiaries with their own healthcare data"
```

File: _projects/0400_blue-button-2.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ banner_file_mobile: banner--people-projects-sm.svg
...
+carousel_summary: "USDS has worked with CMS to empower beneficiaries with their own health data. Leveraging CMS APIs, providers can view of their patients’ full healthcare history without patient intervention. Providers save valuable time that they can better spend talking to patients and preventing conflicts or gaps in care."
```
Comment (lisac-usds) [inline _projects/0400_blue-button-2.md]: noticed this difference btwn the wording in `carousel_title` vs `carousel_summary`:
"...their own healthcare data" vs  "...their own health data". Would it be appropriate to use the same phrase?

File: _projects/0600_ditap.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - figure: 4
+    unit: "x"
+    description: |-
+      Growth of the Digital Acquisition community since 2016, demonstrating the scale that USDS and OFPP envisioned when the program was created
+
+---
+
+## The Challenge
+
+Current federal procurement practices rarely provide the flexibility required to buy and deliver modern digital services. Government’s use of digital solutions has not kept pace with peoples’ expectations. As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, the Office of Federal Procurement Policy (OFPP) and USDS needed an immersive development and training program to improve digital services acquisition expertise across government agencies.
```
Comment (lisac-usds) [inline _projects/0600_ditap.md]: the pace of people's expectations is called out in sentences 2 and 3; should we combine? Maybe we could remove sentence 2?

File: _projects/0600_ditap.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## The Solution
+
+In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation to better acquire today’s technology and works seamlessly with digital service teams or acquisition innovation labs. Graduates of the DITAP program can also lead agency training workshops and provide consultations, expanding digital service procurement expertise across the government. In a [May 2018 memo](whitehouse.gov/wp-content/uploads/2018/05/fac_c_digital_services_05_12_18.pdf), OMB determined that beginning in FY 2022, contracting professionals assigned to digital services acquisitions above $7 million will be required to have FAC‑C‑DS certification.
```
Comment (lisac-usds) [inline _projects/0600_ditap.md]:
```suggestion
In 2016, USDS and OFPP created the Digital IT Acquisition Professional Program (DITAP), a Federal Acquisition Certification in Contracting (FAC‑C) certification program that trains contracting professionals to execute digital service procurements by acting as business advisors to agencies seeking to buy better digital services. DITAP leverages flexibilities in the Federal Acquisition Regulation to better acquire today’s technology and works seamlessly with digital service teams or acquisition innovation labs. Graduates of the DITAP program can also lead agency training workshops and provide consultations, expanding digital service procurement expertise across the government. In a [May 2018 memo](https://whitehouse.gov/wp-content/uploads/2018/05/fac_c_digital_services_05_12_18.pdf), OMB determined that beginning in FY 2022, contracting professionals assigned to digital services acquisitions above $7 million will be required to have FAC‑C‑DS certification.
```
(updated the url to include `https://` prefix; else, it'll be treated like a relative url on the usds.gov website)

**Comment by LauraUSDS** [comment]
@lisac-usds Yes, if we could make that language change for the stats sidebar (per above), that would be great!

**Comment by LauraUSDS** [comment]
Alright! I've proofread everything, checked formatting, caught some typos, made the sidebar header change like Lisa showed me, and.....all looks good now...so I'm going to push live. 🤞🏻

**Comment by LauraUSDS** [comment]
Blast - I just got a 'page build failure' notification. Lisa and Drew, I'll ping y'all next week to see when you might have time to help me investigate!

**Comment by lisac-usds** [comment]
investigating!

**Comment by lisac-usds** [comment]
i wonder if the GitHub pages deployment failure could be due to our config referencing an older version of `jekyll-feed`. nothing jumps out at me for why we can't use the more recent version (0.13 -> 0.15). change proposed in #596.

**Pull request merged**